pkgs/tools/package-management/nix/modular/src/libutil/package.nix

```diff
@@ -23,12 +24,14 @@ mkMesonLibrary (finalAttrs: {
 
   workDir = ./.;
 
+  nativeBuildInputs = [ pkg-config ];
+
   buildInputs =
     [
       brotli
     ]
     ++ lib.optional (lib.versionAtLeast version "2.27") [
-      libblake3
+      (lib.getDev libblake3)
```

Comment:
isn't this automatic since it's in `buildInputs`?

Reply:
I thought so, too but it somehow fails. Maybe splicing is somewhere off?

Reply:
It somehow works for brotli, no idea what is different to libblake3. I tried adding the lib output but no change.

Doing ``nix-shell -A nixVersions.nix_2_29.libs.nix-util`` reveals that the buildInput is build wrong ``buildInputs=/nix/store/slqvy0fgnwmvaq3bxmrvqclph8x909i2-brotli-1.1.0-dev /nix/store/37axr9bpv45nl0hya4pk1kdqsblyhm7f-libblake3-1.8.2 /nix/store/f8n0pjndsxh76x7sv9n5x397rlrgx24p-libsodium-1.0.20-dev /nix/store/kcgqglb4iax0zh5jlrxmjdik93wlgsrq-openssl-3.4.1-dev /nix/store/fdnys4hjcm28i40b7gxi2whzkvfbhjjm-libcpuid-0.8.0``